### DISCUSSION TITLE
Color change for fully constrained elements

### DISCUSSION THREAD

**Discussion by exegetor** [hlorus/CAD_Sketcher#233] [Feature Requests]
Per request on discord regarding difficulty identifying location of degrees of freedom:

The idea is that when an elements is fully constrained, it is rendered differently so user can tell it's "locked down" and not a source of any degrees of freedom.

An alternative to color change might be displaying thicker line width.

**Comment by hlorus** [comment]
This is simply a limitation of the solver. Afaik solvespace itself also doesn't communicate this per element. So maybe there's already a topic on their forum somewhere but i haven't found that yet.

**Comment by SiboVG** [comment]
@hlorus this appears to be a duplicate of #61, should either one be closed?

**Comment by hlorus** [comment]
Looks like it's not possible to merge discussions, i'll just lock it then.
